runtime/metrics.rs

```diff
@@ -1,16 +1,30 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 use deno_core::serde::Serialize;
+use deno_core::Op;
+use deno_core::OpFn;
+use deno_core::OpId;
 
-#[derive(Default, Debug)]
+#[derive(Debug)]
 pub struct RuntimeMetrics {
-  pub ops: HashMap<&'static str, OpMetrics>,
+  max_id: OpId,
+  pub ops: Vec<OpMetrics>,
+}
+
+impl Default for RuntimeMetrics {
+  fn default() -> Self {
+    // 256 is 2^8 and big enough for all of our ops
+    Self {
+      max_id: 0,
+      ops: vec![OpMetrics::default(); 256],
```

Comment:
since it's fixed size, does it need to be a vec! ?

Reply:
It could in theory grow dynamically as ops are registered via plugins (plugins won't work in 1.9 but will return in 1.10).